cpp/CMakeLists.txt

```diff
@@ -50,3 +50,4 @@ endfunction()
 # Example usage
 create_target(hello_world "src/hello_world.cpp")
 create_target(send_recv "src/send_recv.cpp")
+create_target(mpi_reduce "src/mpi_reduce.cpp")
```

Comment:
Let's just call it reduce to keep the naming consistent as we are just using MPI.

Reply:
file name & its references renamed to reduce.cpp